README.md

```diff
@@ -110,6 +110,15 @@
       * `.config/kaccessrc`
    * Applications
 	   * File associations
+       		* System-wide default settings:
+    			* `/usr/share/applications/mimeapps.list`
+    			* `/usr/share/applications/kde-mimeapps.list`
+       		* System-wide custom settings:
+    			* `/etc/xdg/mimeapps.list`
+    			* `/etc/xdg/kde-mimeapps.list`
+        	* Users' custom settings:
+           		* `.config/mimeapps.list`
+    			* `.config/kde-mimeapps.list`
```

Comment:
Thanks for PR. I would suggest to only keep user's settings paths. Otherwise there would be too much stuff in a single readme file.

```suggestion
           * `.config/mimeapps.list`
           * `.config/kde-mimeapps.list`
```